### ISSUE TITLE
Crash at PlayerStorage.cpp:7361

### ISSUE THREAD

**Issue by FrancescoBorzi** [azerothcore/azerothcore-wotlk#8273]
```
Thread 25 "worldserver" received signal SIGSEGV, Segmentation fault.
[Switching to Thread 0x1526693fe700 (LWP 32498)]
Player::_SaveInventory (this=<optimized out>, trans=...) at /root/env/chromiecraft/azerothcore-wotlk/src/server/game/Entities/Player/PlayerStorage.cpp:7361
7361	        Bag* container = item->GetContainer();
#0  Player::_SaveInventory (this=<optimized out>, trans=...) at /root/env/chromiecraft/azerothcore-wotlk/src/server/game/Entities/Player/PlayerStorage.cpp:7361
#1  0x0000000000d38a3e in Player::SaveToDB (this=0x1526c0568540, trans=warning: 121958.830174 RTTI symbol not found for class 'std::_Sp_counted_ptr_inplace<Transaction<CharacterDatabaseConnection>, std::allocator<Transaction<CharacterDatabaseConnection> >, (__gnu_cxx::_Lock_policy)2>'
121958.830177 
warning: 121958.830320 RTTI symbol not found for class 'std::_Sp_counted_ptr_inplace<Transaction<CharacterDatabaseConnection>, std::allocator<Transaction<CharacterDatabaseConnection> >, (__gnu_cxx::_Lock_policy)2>'
121958.830321 
std::shared_ptr<Transaction<CharacterDatabaseConnection>> (use count 3, weak count 0) = {...}, create=false, logout=<optimized out>) at /root/env/chromiecraft/azerothcore-wotlk/src/server/game/Entities/Player/PlayerStorage.cpp:7129
#2  0x0000000000d383dd in Player::SaveToDB (this=0x1526c0568540, create=false, logout=false) at /root/env/chromiecraft/azerothcore-wotlk/src/server/game/Entities/Player/PlayerStorage.cpp:7093
#3  0x0000000000d52371 in Player::Update (this=<optimized out>, p_time=10) at /root/env/chromiecraft/azerothcore-wotlk/src/server/game/Entities/Player/PlayerUpdates.cpp:302
#4  0x0000000000ff6123 in Map::Update (this=0x15268b83dcc0, t_diff=0, s_diff=10) at /root/env/chromiecraft/azerothcore-wotlk/src/server/game/Maps/Map.cpp:746
#5  0x000000000101268c in MapUpdateRequest::call (this=0x1523914a5600) at /root/env/chromiecraft/azerothcore-wotlk/src/server/game/Maps/MapUpdater.cpp:31
#6  0x000000000101201a in MapUpdater::WorkerThread (this=0x1ce3010 <MapMgr::instance()::instance+216>) at /root/env/chromiecraft/azerothcore-wotlk/src/server/game/Maps/MapUpdater.cpp:146
#7  0x00001526c9e9ede4 in ?? () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#8  0x00001526c9fb9609 in start_thread (arg=<optimized out>) at pthread_create.c:477
#9  0x00001526c9b8e293 in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:95
```

### AC VERSION

https://github.com/chromiecraft/azerothcore-wotlk/commit/b2f08934bcc60491f8fa8971f98837b2d48a489e

##### OPERATING SYSTEM:

Ubuntu 20.04

##### MODULES:

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-CarbonCopy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-LevelUpReward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-Zonecheck](https://github.com/55Honey/acore_Zonecheck)

##### OTHER CUSTOMIZATIONS:

None.

##### SERVER:

ChromieCraft

**Comment by Nefertumm** [comment]
Any idea how to reproduce this?

**Comment by Winfidonarleyan** [comment]
Welcome pets queries

**Comment by Nefertumm** [comment]
@TempName55555 stop.

**Comment by Nefertumm** [comment]
> Welcome pet queries

pet queries no :(

**Comment by Nefertumm** [comment]
Closed by 702573038e89d2c7f446fad467261c75b9f90f82
